examples/04_manipulating_images/plot_nifti_simple.py

```diff
@@ -20,8 +20,9 @@
 # Compute the mask
 from nilearn.input_data import NiftiMasker
 
-# As this is raw resting-state EPI, the background is noisy and we cannot
-# rely on the 'background' masking strategy. We need to use the 'epi' one
+# As this is raw movie watching based EPI, the background is noisy and we
```

Comment:
So this is an actual comment --  we've already masked this data using a brain mask. Should we update this example, or leave as is ? Or do you want me to update the data ?